itou/job_applications/models.py

```diff
@@ -503,19 +498,19 @@ def accepted_by(self):
 
     @property
     def can_download_approval_as_pdf(self):
-        return (
-            self.state.is_accepted
-            and not self.can_be_cancelled
-            and self.to_siae.is_subject_to_eligibility_rules
-            and self.approval
-        )
+        return self.state.is_accepted and self.to_siae.is_subject_to_eligibility_rules and self.approval
 
     @property
     def can_be_cancelled(self):
         if self.hiring_start_at:
-            today = datetime.date.today()
-            delay_ends_at = self.hiring_start_at + relativedelta(days=self.CANCELLATION_DAYS_AFTER_HIRING_STARTED)
-            return today <= delay_ends_at
+            # A job application can be cancelled provided that
+            # there is no employee record linked with a status :
```

Comment:
```suggestion
            # there is no employee record linked with a status:
```

Pas d'espace avant les `:` en anglais ;)